gds/src/main/scala/edu/gemini/aspen/gds/util/Enumerated.scala

```diff
@@ -0,0 +1,48 @@
+package edu.gemini.aspen.gds.util
+
+import cats.Order
+import cats.syntax.all._
+
+/**
+ * Typeclass for an enumerated type with unique string tags and a canonical ordering.
+ * @group Typeclasses
+ */
+trait Enumerated[A] extends Order[A] {
```

Comment:
Should we use the one from `lucuma-core`? I guess OSGI could make it difficult

Reply:
As above, I was trying to limit dependencies somewhat. `lucuma-core` would transitively pull a lot more dependencies into the `gds-fp` bundle. I decided since all I wanted was a reasonable Enum representation, and scala 3 was out of scope, to just copy this in and remove the bits we didn't need - the the circe en/decoders.